src/style/modules/_basestructure.scss

```
@@ -408,18 +408,30 @@ div.content.custom > main > section > div > div > div > h3 {
     box-shadow: 0 2px 3px rgba(34, 34, 34, 0.1), 0 0 0 1px rgba(34, 34, 34, 0.1);
     margin-top: 22px;
     font-size: 17px;
+    display: flex;
     @media (min-width: 769px) {
-      height: 267px;
       padding-bottom: 10%;
       padding-top: 10%;
     }
     left: 50%;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)